content/guides/sync.md

```diff
@@ -68,74 +70,76 @@ This section, together with a [DEFAULT] section, constitutes the full path of av
+Job folders for each customer are placed under the same `EGI/` directory and names are read from config file. Segment of configuration file that reflects the creation of directories is for example: 
 
-	[CUSTOMER]
+	[DIR]
+	OutputDir = /var/lib/argo-connectors/EGI/
+
+	[CUSTOMER_EGI]
 	Jobs = JOB_Test1, JOB_Test2
 
 	[JOB_Test1]
-	Dirname = Testing1
+	Dirname = EGI_Testing1
 
 	[JOB_Test2]
-	Dirname = Testing2
+	Dirname = EGI_Testing2
 
-	[VO_VOTesting]
-	Dirname = VoName
-	VOFeed = http://kosjenka.srce.hr/~eimamagi/ops.feed.xml
-	Jobs = JOB_VOTest1, JOB_VOTest2
 
-	[JOB_VOTest1]
-	Dirname = Testing1
+	[CUSTOMER_NGI]
+	Jobs = Job_Test3, JOB_Test4
 
-	[JOB_VOTest2]
-	Dirname = Testing2
+	[JOB_Test3]
+	Dirname = NGI_Testing1
 
-This will result in the following directory structure:
+	[JOB_Test4]
+	Dirname = NGI_Testing2
 
-	/EGI/Testing1
-	/EGI/Testing2
-	/EGI/VoName/Testing1
-	/EGI/VoName/Testing2
+This will result in the following jobs directories:
 
-So there is EGI as entity, identified with `[CUSTOMER]` section, and one VO VOTesting identified with `[VO_VOTesting]` section and accompanied by `VOFeed`. Multiple VOes can be specified. Each entity have set of jobs listed in `Jobs` option and entity can not exist without associated jobs. The name of the job folder is specified with `Dirname` option of the certain job. JOB\_Test1, identified with `[JOB_Test1]` section, will be named Testing1 and since JOB\_Test1 belongs to EGI, it will be placed under EGI/ directory. JOB\_VOTest2 will be named Testing2, but since it belongs to VOTesting it will be placed under VO's directory specified with `Dirname` option, which is VoName/.
+	/var/lib/argo-connectors/EGI/EGI_Testing1
+	/var/lib/argo-connectors/EGI/EGI_Testing2
+	/var/lib/argo-connectors/EGI/NGI_Testing1
+	/var/lib/argo-connectors/EGI/NGI_Testing2
 
-Every connector reads this configuration file because it needs to find out the directory structure where to lay down its files. So `poem-connector.py`, `downtimes-gocdb-connector.py`, `weights-gstat-connector.py`, all of them are writing theirs data in each job folder for EGI and each job folder for each given VO. Topology distinguishes for different entity so exceptions to this are `topology-gocdb-connector.py` and `topology-vo-connector.py`. They are writing data for one entity, EGI or for each given VO, respectively.
+So there are two customers, EGI and NGI, each one identified with its `[CUSTOMER_*]` section. `CUSTOMER_` is a section keyword and must be specified when one wants to define a new customer. Each customer has set of jobs listed in `Jobs` option and customer can not exist without associated jobs. The name of the job folder is specified with `Dirname` option of the certain job so JOB\_Test1, identified with `[JOB_Test1]` section, will be named EGI\_Testing1 and it will be placed under EGI/ directory. 
```

Comment:
I think maybe JOB\_Test1 should be replaced with `JOB\_Test1`. Several other similar changes are needed in this line.